### PR TITLE
Auto-build Docker image

### PR DESCRIPTION
**Pull request by duckinator**

### CODE REVIEW

**Comment by duckinator** [comment]
Current release process:

```
$ docker build -t duckinator/bork:$VERSION --build-arg VERSION=$VERSION
$ docker tag duckinator/bork:$VERSION duckinator/bork:latest # If appropriate.
$ docker push duckinator/bork:$VERSION
$ docker push duckinator/bork:latest # If appropriate.
```

I want to automate this. Will poke at it more either later tonight or sometime tomorrow.

**Comment by duckinator** [comment]
cc @astronouth7303

**Comment by astronouth7303** [comment]
You can do it with cirrus (KVM container or docker builder). Docker hub also has a basic CD feature.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-slim
+
+ARG VERSION
```
Comment (astronouth7303) [inline Dockerfile:3]: Is there a default you can set this to, so that if it's not given it'll just use the latest?

Comment (duckinator) [reply Dockerfile:3]: The way to do that is `ARG=<default value>`, but I need to find a value that'll tell pip to install the latest available version.

Comment (duckinator) [reply Dockerfile:3]: Could maybe do this?:

```
ARG VERSION=""

RUN [ -z "$VERSION" ] && pip3 install bork || pip3 install "bork==${VERSION}"
```

but idk how i feel about that.

Comment (duckinator) [reply Dockerfile:3]: A hacky "works for now i guess" type of approach would be to just hard-code the latest released version in the Dockerfile and bump it with setup.cfg.

Comment (astronouth7303) [reply Dockerfile:3]: Reasons I wrote buildahscript

**Comment by duckinator** [comment]
@astronouth7303 I set up Cirrus CI to build an image and push it to Docker Hub whenever a tag is pushed to GitHub. I want an option to have `bork release` push a new tag via git, so this is incentive for me to do that, I guess. :stuck_out_tongue:

**Comment by astronouth7303** [comment]
yay!

**Pull request merged**